hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/scm/container/common/helpers/DeletedBlocksTransactionInfoWrapper.java

```diff
@@ -38,6 +38,12 @@ public DeletedBlocksTransactionInfoWrapper(long txID, long containerID,
     this.localIdList = localIdList;
     this.count = count;
   }
+  public DeletedBlocksTransactionInfoWrapper() {
```

Comment:
6883aee5ee02825d356ac9a8050312eafdb1b69a removed `ResetDeletedBlockRetryCountSubcommand` from the PR, maybe accidentally.  This new `DeletedBlocksTransactionInfoWrapper` constructor belongs to that part.